ega-charts/localega/values.yaml

```diff
@@ -290,5 +296,5 @@ tester:
   log: "debug"
   size: "small"
   repository: nbisweden/localega-tester
-  imageTag: m7
+  imageTag: m7-v2
```

Comment:
we updated tester, we should be able to use latest.

Reply:
The m7-v2 tag is build from the last tester PR, better to use a fixed tag than master.

Reply:
ok